pkg/pillar/cmd/nodeagent/handletimers.go

```diff
@@ -322,5 +322,13 @@ func handleNodeReboot(ctxPtr *nodeagentContext) {
 	log.Functionf("Timer Expired.. Zboot.Reset()")
 	syscall.Sync()
 	<-rebootTimer.C
+	rebootTimer = time.NewTimer(duration)
```

Comment:
Reusing 	duration := time.Second * time.Duration(minRebootDelay) seems a bit odd. But I don't know if it would be better to use a fixed timer (10 seconds??). Thoughts?

Reply:
I put 20 seconds of delay, because we have 10 seconds of waiting inside zboot https://github.com/lf-edge/eve/blob/a4d135b5667d0b4f73e9f6c9626d1f24d71c4a98/pkg/pillar/zboot/zboot.go#L86-L87 and it is reasonable to give zboot ability to retry.

Reply:
Makes sense (even if in this hang case it will not make any difference AFAICT due to the unkillable wait in the kernel).